CMakeLists.txt

```diff
@@ -43,1343 +88,243 @@ add_executable(
   $<${is-posix}:${D}/src/d8/d8-posix.cc>
   $<${is-win}:${D}/src/d8/d8-windows.cc>
   ${D}/src/d8/async-hooks-wrapper.cc
-  ${D}/src/d8/async-hooks-wrapper.h
```

Comment:
I have two reasons for including header files:

1. I'm informed that it produces nicer VS project files
2. It makes syncing with upstream's GN files easier (because they include header files in source lists)

Unless there's a strong reason to leave them out, I'd rather leave them in.

Reply:
It reduces some configuration and generation time. (It can also clash with the automatic dependencies that compilers produce in ninja). There's a different path to take when creating visual studio project files:

1) Set `USE_FOLDERS` property to on
2) Use the [`source_group`][1] command to add the headers to IDEs only

Alternatively
Recommend people use the CMake + Ninja setup that Visual Studio supports :P

[1]: https://cmake.org/cmake/help/latest/command/source_group.html